Source/JavaScriptCore/runtime/IntlLocale.cpp

```diff
@@ -643,7 +643,7 @@ JSArray* IntlLocale::hourCycles(JSGlobalObject* globalObject)
         return nullptr;
     }
 
-    dataLogLnIf(IntlLocaleInternal::verbose, "pattern:(", StringView(pattern.data(), pattern.size()), ")");
+    dataLogLnIf(IntlLocaleInternal::verbose, "pattern:(", StringView(pattern.span()), ")");
```

Comment:
Can use `span` directly here, I think. No need for `StringView`.

Reply:
This one didn't work.